engine/app/components/citizens_advice_components/footer.html.erb

```diff
@@ -4,12 +4,22 @@
       <div class="cads-grid-row">
         <div class="cads-grid-col">
           <p class="cads-footer__feedback">
-            <%= link_to(
-              feedback_link.title || t("citizens_advice_components.footer.website_feedback"),
-              feedback_link.url,
-              class: "cads-footer__hyperlink js-cads-footer-feedback",
-              **feedback_link.new_tab_attributes
-            ) %>
+            <% if feedback_link.new_tab? %>
```

Comment:
I wanted to fix the issue just by adding a conditional for the aria-label in the `.rb` file. However, I cannot use translations there and was getting error: https://www.rubydoc.info/gems/view_component/3.11.0/ViewComponent/TranslateCalledBeforeRenderError. So instead I decided to expose the new_tab_attributes here.
Suggestions for a more elegant solution are welcome.

Reply:
Yeah that rings a bell, I think what you've done here _is_ the elegant solution as it's part of the template logic anyway.